src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public Bulldozer(String name) {
+        super(name);
+    }
+
+    @Override
+    public void doWork() {
+        System.out.println(super.getName() + " Bulldozer started working");
```

Comment:
If you want to get class name you can use `getClass().getSimpleName()`.
So, you do not need to have name property in `Machine` class